src/processTargets/targets/PositionTarget.ts

```diff
@@ -102,37 +105,23 @@ export default class PositionTarget extends BaseTarget {
   }
 
   private getEditText(text: string) {
-    if (this.isLineDelimiter) {
-      return this.isBefore
-        ? this.linePadding + text + this.insertionDelimiter
-        : this.insertionDelimiter + this.linePadding + text;
-    }
+    const insertionText = this.indentationString + text;
+
     return this.isBefore
-      ? text + this.insertionDelimiter
-      : this.insertionDelimiter + text;
+      ? insertionText + this.insertionDelimiter
+      : this.insertionDelimiter + insertionText;
   }
 
   private updateRange(range: Range, text: string) {
-    const startIndex = (() => {
-      if (this.isLineDelimiter) {
-        const line = this.editor.document.lineAt(
-          this.isBefore ? range.end : range.start
-        );
-        const startOffset = this.editor.document.offsetAt(
-          this.isBefore ? line.range.start : line.range.end
-        );
-        return this.isBefore
-          ? startOffset - this.insertionDelimiter.length - text.length
-          : startOffset +
-              this.insertionDelimiter.length +
-              this.linePadding.length;
-      }
-      const startOffset = this.editor.document.offsetAt(range.start);
-      return this.isBefore
-        ? startOffset
-        : startOffset + this.insertionDelimiter.length;
-    })();
+    const baseStartOffset = this.editor.document.offsetAt(range.start);
+    const startIndex = this.isBefore
+      ? baseStartOffset + this.indentationString.length
+      : baseStartOffset +
+        this.insertionDelimiter.length +
+        this.indentationString.length;
```

Comment:
As above, we don't need the special casing, because `indentationString` is just empty when `!this.isLineDelimiter`